<launch>
 <node name="joy2" pkg="joy" type="joy_node" output="screen">
    <param name="dev" value="/dev/input/js0"/>
    <param name="default_trig_val" value="true"/>
 </node>
 
 <node name="controller" pkg="gen2bot" type="notDTTalker.py" output="screen">
 </node>

 <node name="wheelies" pkg="gen2bot" type="listenerMotor" output="screen">
 </node>

<!-- Launches my listenerMotorAuto.cpp file to initialize a base controller 
  that move_base communicates with to send motor inputs. This links autonomy to motors -->
 <node pkg="gen2bot" type="notDT" name="notDT" output="screen">

    <param name="wheel_multiplier" value="1.1" type="double"/>

    <rosparam file="$(find gen2bot)/config/PID.yaml" command="load"/> 
 </node>

</launch>
